test/e2e/load.go

```diff
@@ -139,18 +154,18 @@ func computeRCCounts(total int) (int, int, int) {
 	return smallRCCount, mediumRCCount, bigRCCount
 }
 
-func generateRCConfigs(totalPods int, c *client.Client, ns string) []*RCConfig {
+func generateRCConfigs(totalPods int, image string, command []string, c *client.Client, ns string) []*RCConfig {
```

Comment:
Does this approach still make sense with such a small number of pods (for the 3 pods/node case)? Maybe just split pods evenly among 3 RCs or something like that?